packages/core/src/textures/Texture.ts

```diff
@@ -16,6 +16,19 @@ const DEFAULT_UVS = new TextureUvs();
 
 export type TextureSource = string | BaseTexture | ImageSource;
 
+/** INineSliceBorders stores the width of the non-scalable borders of a {@link PIXI.NineSlicePlane} texture */
+export interface INineSliceBorders
```

Comment:
Preference here would be to call this `ITextureBorders` instead of `INineSliceBorders`. This shouldn't be couple so close with NineSplicePlane, though I understand that's the primary use-case.

```suggestion
/** Stores the width of the non-scalable borders, for example when used with {@link PIXI.NineSlicePlane} texture */
export interface ITextureBorders
```